fearless/Modules/WalletMainContainer/WalletMainContainerViewLayout.swift

```diff
@@ -72,19 +73,26 @@ final class WalletMainContainerViewLayout: UIView {
 
     // MARK: - Address label
 
-    private let addressCopyableLabel: CopyableLabelView = {
-        let label = CopyableLabelView()
-        return label
+    private let addressCopyableLabel = CopyableLabelView()
+
+    // MARK: - Ussues button
+
+    let issuesButton: UIButton = {
+        let button = UIButton()
+        button.semanticContentAttribute = .forceRightToLeft
+        button.setImage(R.image.iconWarning(), for: .normal)
+        button.setTitle("Network Issues", for: .normal)
+        button.titleLabel?.font = .h6Title
+        button.layer.masksToBounds = true
+        button.backgroundColor = R.color.colorWhite8()
+        button.layer.cornerRadius = Constants.issuesButtonSize.height / 2
+        button.textAndImage(spacing: 5)
+        return button
     }()
 
     // MARK: - FWSegmentedControl
 
-    let segmentedControl: FWSegmentedControl = {
-        let segment = FWSegmentedControl()
-        let items = ["Currencies", "NFTs"]
-        segment.setSegmentItems(items)
-        return segment
-    }()
+    let segmentedControl = FWSegmentedControl()
```

Comment:
why? don't you miss titles ?

Reply:
localized title. Set on applyLocalization method